### ISSUE TITLE
ENCRYPTION_KEY_DEV on production.

### ISSUE THREAD

**Issue by Tkd-Alex** [chartbrew/chartbrew#216]
https://github.com/chartbrew/chartbrew/blob/04b923c305484ce91b179418266dd9bffe51dd61/server/modules/cbCrypto.js#L4

Hello, I'm running chartbrew via Docker.
I'm not sure it's correct to instantiate `SECRET_KEY` using `CB_ENCRYPTION_KEY_DEV`, should be `CB_ENCRYPTION_KEY`.

Currently, 
after upgrade to the latest version I'm unable to login to chartbrew.

**Comment by razvanilin** [comment]
Hey @Tkd-Alex that's a good spot! Not sure how it worked on my end, but that should select the production var instead of the DEV one indeed.

I fixed that, but need to run a few more tests to make sure it doesn't break anything else. Will update shortly

**Comment by razvanilin** [comment]
I just released [v3.4.0](https://github.com/chartbrew/chartbrew/releases/tag/v3.4.0) that fixes the issue. I also added some tips on how proceed if you already used Chartbrew 3.3.0 to create connections, datasets, charts, etc. Essentially, you will need to replace `CB_ENCRYPTION_KEY` value with the one from `CB_ENCRYPTION_KEY_DEV`. If you use Docker, you will have to inspect the files in the container and check the value of `CB_ENCRYPTION_KEY_DEV` in the `.env` file

I'll close this issue, but feel free to open again if there are any issues
